project/dagger-query.sh

```diff
@@ -0,0 +1,7 @@
+export PATH="$PATH:$HOME/bin"
+
+if [[ -z $3 ]]; then
```

Comment:
Would it be possible to pass through user arguments with `$@`?

Reply:
Yeah, that works! Thanks, I didn't know about it before.